### PR TITLE
MALTD: 403/404

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Description

This PR adds the basic list element base component and story. Jira tickets:
- https://justice.gov.bc.ca/jira/browse/MDYNAMICS-403
- https://justice.gov.bc.ca/jira/browse/MDYNAMICS-404

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit tests, and visually through storybook

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: maltd-frontend/src/components/base/ListElement/ListElement.stories.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import ListElement from "./ListElement";
+
+const listElement = {
+  title: "MALTD",
```
Comment (akroon3r) [inline maltd-frontend/src/components/base/ListElement/ListElement.stories.js]: maybe a TODO to remove this at a later date? 😝

File: maltd-frontend/src/components/base/ListElement/ListElement.stories.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+storiesOf("List Element", module)
+  .add("current projects", () => <ListElement listElement={listElement} />)
+  .add("users", () => (
+    <ListElement
+      listElement={{
+        ...listElement,
+        title: "Pat Simonson",
+        description: "psimon@gov.bc.ca"
```
Comment (akroon3r) [inline maltd-frontend/src/components/base/ListElement/ListElement.stories.js]: I know this is dummy data, but are there any issues with this being in the repo if it did actually exist? It already exists in the PR history now.

Dumb question clearly.

**Comment by akroon3r** [comment]
Small things, looks clean though.

File: maltd-frontend/src/components/base/ListElement/ListElement.stories.js (hunk 1 above)
Comment (sdevalapurkar) [reply maltd-frontend/src/components/base/ListElement/ListElement.stories.js]: why? it is an open source project right so this placeholder value as a sample project title should be ok right? 😄

Comment (akroon3r) [reply maltd-frontend/src/components/base/ListElement/ListElement.stories.js]: You're not wrong.

File: maltd-frontend/src/components/base/ListElement/ListElement.stories.js (hunk 2 above)
Comment (sdevalapurkar) [reply maltd-frontend/src/components/base/ListElement/ListElement.stories.js]: hmm yeah I didn't check that it doesn't exist - however, I did make it up as dummy data based on the mockups - I will change it to be something more dummy though just as a safety

**Pull request merged**